CuratedContent/LanguageReferenceOnLine.md

```diff
@@ -0,0 +1,67 @@
+# Programming Language Reference Manuals On Line
+<!--deck text start-->
+<!--deck text end-->
+
+#### Contributed by [Mark C. Miller](https://github.com/markcmiller86 "Mark C. Miller GitHub Profile")
+
+### C, C++, C Pre-Processor
+
+* [C Preprocessor](https://gcc.gnu.org/onlinedocs/cpp/)
+* [Microsoft C, C++ and Assembler documentation](https://docs.microsoft.com/en-us/cpp/?view=msvc-170)
+* [C++ Reference](https://en.cppreference.com/w/)
+* [C++ Library Reference](https://www.cplusplus.com/reference/)
+* [cplusplus.com](https://www.cplusplus.com)
+* [C Standard Library Reference](https://www.cplusplus.com/reference/clibrary/)
+* [Posix C Library](https://en.wikipedia.org/wiki/C_POSIX_library)
+* [GNU C Reference Manaul](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html)
+* [GNU C Library Reference Manual](https://www.gnu.org/software/libc/manual/pdf/libc.pdf)
+* [C Language Cheat Sheet](https://users.ece.utexas.edu/~adnan/c-refcard.pdf)
+
+### Fortran
```

Comment:
The final working draft (I think) for F18 is https://j3-fortran.org/doc/year/18/18-007r1.pdf.

Overall, I would say F77, F95, F08 and the latest (F18) might be primary points of interest in Fortran history.

Reply:
My standards were always 77, 95 and 2003 but thats just me.

That said, for fortran, I am wondering whether its better to refer to specific compiler vendors language specs instead of any standards committee? Reason being is that the compiler vendors specs define what has been implemented which is often not the complete standard specification.